aos/closed_loop/calcZernikesFromDonutStamps.ipynb

```diff
@@ -8,10 +8,10 @@
     "# CalcZernikes from DonutStamps\n",
```

Comment:
<p>I'm thinking of a future reader wondering why  the same dataId's are used here. Worth adding just before this cell that the fact that both stamp sets use the same extra-focal dataId is not a typo - they are both stored under the same extra-focal dataId, and using intra-focal dataId to get <code>dataStampsIntra</code> wouldn't work at all.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Ah good point, JF already asked me about this a couple days ago. I'll add the explanation here.</p>